fortanix-vme/nitro-attestation-verify/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "nitro-attestation-verify"
+version = "0.1.0"
+authors = ["Adrian Cruceru <adrian.cruceru@fortanix.com>"]
+edition = "2018"
+
+[dependencies]
+serde_cbor = {version = "0.11" }
+aws-nitro-enclaves-cose = { git = "https://github.com/fortanix/aws-nitro-enclaves-cose.git", branch = "raoul/crypto_abstraction", default-features = false }
+mbedtls8 = { git = "https://github.com/fortanix/rust-mbedtls", package = "mbedtls", branch = "acruceru/port-async-support", features = ["rdrand", "std", "async-rt", "dsa", "time"], default-features = false  }
```

Comment:
perhaps this needs to be a release branch or something? (since code review there has been going on for a while)

A force push would cause all builds to fail everywhere.

Reply:
same comment for aws-nitro-enclaves-cose

Reply:
Ah yes the release branch of `mbedtls` should work here. For `aws-nitro-enclaves-cose`, I created a PR: https://github.com/awslabs/aws-nitro-enclaves-cose/pull/36

Reply:
Will also wait until the [mbedtls PR](https://github.com/fortanix/rust-mbedtls/pull/172) merged

Reply:
The [mbedtls PR](https://github.com/fortanix/rust-mbedtls/pull/172) merged. The [aws-nitro-enclaves-cose PR](https://github.com/awslabs/aws-nitro-enclaves-cose/pull/36) is talking longer than I was hoping for. I used a separate branch to avoid that a forced push for the [aws-nitro-enclaves-cose PR](https://github.com/awslabs/aws-nitro-enclaves-cose/pull/36) inadvertently breaks the `Cargo.lock` file of this repo